code/game/turfs/simulated/floor/plasteel_floor.dm

```diff
@@ -25,9 +25,24 @@
 	return
 
 /turf/simulated/floor/plasteel/goonplaque
-	icon_state = "plaque"
 	name = "Commemorative Plaque"
 	desc = "\"This is a plaque in honour of our comrades on the G4407 Stations. Hopefully TG4407 model can live up to your fame and fortune.\" Scratched in beneath that is a crude image of a meteor and a spaceman. The spaceman is laughing. The meteor is exploding."
+	icon_state = "plaque"
+
+/turf/simulated/floor/plasteel/goonplaque/memorial
+	name = "Memorial Plaque"
+	desc = "\"This is a plaque in honour of those who died in the great space lube airlock incident.\" Scratched in beneath that is a crude image of a clown and a spaceman. The spaceman is slipping. The clown is laughing."
+
+/turf/simulated/floor/plasteel/goonplaque/commission
+	name = "Commission Plaque"
+	desc = "Epsilon Eridani Sector - 'Meta' Class Outpost - Commissioned 11/03/2557 - NSS Cerebron"
+
+/turf/simulated/floor/plasteel/goonplaque/nosey
+	name = "Nosey little bastard aren't you?"
+	desc = "Nosey little bastard aren't you?"
```

Comment:
my easter egg, exposed to all!

Reply:
I tried to find it without cheating, failed.